### PR TITLE
[fix] format metrics keys appropriately

### PR DESCRIPTION
**Pull request by VkoHov**
1. Encode metric key by name and context 
2. Added logic for creating valid queries if there are double quotes inside the metric name
3. Create `getLabelAndValueOfMetrics` method and replace it everywhere
4. Fix sorting from the table middle panel columns
5. Fix export table keys converting
6. Adjust getFilteredRow method

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#2005](https://github.com/aimhubio/aim/issues/2005)

**Review by rubenaprikyan — APPROVED** [review]
Lgtm!

File: aim/web/ui/src/modules/BaseExplorer/components/Figures/Figures.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ function Figures(props: any) {
         }
       }
     }
-  }, [data, props.style?.width, props.style?.height]);
+  }, [data, props.style.width, props.style.height, props.style]);
```
Comment (roubkar) [inline aim/web/ui/src/modules/BaseExplorer/components/Figures/Figures.tsx:39]: please check whether `props.style` is being frequently changed

File: aim/web/ui/src/utils/app/getFilteredRow.ts [hunk 2]
```diff
@@ -6,17 +11,24 @@ export default function getFilteredRow<R extends Record<string, any>>({
   row: R;
 }): { [key: string]: string } {
   return columnKeys.reduce((acc: { [key: string]: string }, column: string) => {
+    let columnKey = column;
+    if (isMetricHash(column)) {
```
Comment (roubkar) [inline aim/web/ui/src/utils/app/getFilteredRow.ts:15]: please use utility functions for metric hash and label

File: aim/web/ui/src/utils/app/getSelectOptions.ts [hunk 3]
```diff
@@ -1,16 +1,20 @@
+import _ from 'lodash';
```
Comment (roubkar) [inline aim/web/ui/src/utils/app/getSelectOptions.ts:1]: please use "lodash-es"

**Review by roubkar — APPROVED** [review]

**Pull request merged**